charts/kubescape-operator/values.yaml

```diff
@@ -163,6 +163,31 @@ global:
   networkPolicy:
     enabled: false
     createEgressRules: false
+    commonEgressRules:
+      - ports:
+          - port: 53
+            protocol: UDP
+        to:
+          - namespaceSelector:
+              matchLabels:
+                kubernetes.io/metadata.name: kube-system
+            podSelector:
+              matchLabels:
+                k8s-app: kube-dns
+      - ports:
+          - port: 4317
+            protocol: TCP
+        to:
+          - podSelector:
+              matchLabels:
+                app: otel-collector
+    k8sApiEgressRules:
```

Comment:
the idea is that we list here rules to reach the api-server for well-known distributions (minikube, kind, GKE, ...) but we let users override it to their needs